src/template/Header/Header.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+
+import Preloader from 'ui-kit/preloader/preloader'
+
+import { requestImages } from '../../redux/action'
```

Comment:
the file should be **redux/actions** instead of **redux/action**, because you have in there multiple actions, not only one